CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+<!--lint disable no-duplicate-headings-->
```

Comment:
maybe we want to the same as stac-spec and add a .remarkignore with 

```
/CHANGELOG.md
```
?

Reply:
No, all other rules should still apply.